src/sentry/static/sentry/app/components/dropdownLink.tsx

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import DropdownMenu from 'app/components/dropdownMenu';
+
+// .dropdown-actor-title = flexbox to fix vertical alignment on firefox Need
+// the extra container because dropdown-menu alignment is off if
+// `dropdown-actor` is a flexbox
+
+type Props = Omit<
+  Omit<DropdownMenu['props'], 'children'>,
+  keyof typeof DropdownMenu.defaultProps
+> &
```

Comment:
A little tricky since we're inheriting from a component class which properly supports default prop types.